### PR TITLE
Lakh jobs section

### PR DESCRIPTION
**Pull request by itxjashankapoor**

### CODE REVIEW

File: app/Models/StateTax.php [hunk 1]
```diff
@@ -6,15 +6,19 @@
 use Illuminate\Database\Eloquent\Model;
 
 class StateTax extends Model
-{
+{   
     use HasFactory;
     public $table = "state_taxs";
```
Comment (itxjashankapoor) [inline app/Models/StateTax.php:11]: Please follow naming conventions as directed by Laravel for mapping migrations from Model automatically instaed of defining $table manually in Model. Reference: https://laravel.com/docs/10.x/eloquent#table-names

image

image

In your case migration name should be 'state_taxes'.

File: app/Http/Livewire/Admin/States/State.php [hunk 2]
```diff
@@ -33,8 +32,7 @@ public function store(){
         );
           
         $state = new StateTax;
-        // $state->name = $this->name;
-        $state->code = $this->state_code;
+        $state->state_id = $this->state_code;
```
Comment (itxamit) [inline app/Http/Livewire/Admin/States/State.php:35]:
```suggestion
        $state->state_id = $this->state_id;
```

File: app/Http/Livewire/Admin/States/State.php [hunk 3]
```diff
@@ -16,7 +16,6 @@ class State extends Component
 
       public function resetFields()
     {
-        // $this->name = '';
         $this->state_code = '';
```
Comment (itxamit) [inline app/Http/Livewire/Admin/States/State.php:19]:
```suggestion
        $this->state_id = '';
```

File: app/Http/Livewire/Admin/States/State.php [hunk 4]
```diff
@@ -16,7 +16,6 @@ class State extends Component
 
       public function resetFields()
```
Comment (itxjashankapoor) [inline app/Http/Livewire/Admin/States/State.php:17]:
```suggestion
      public function resetFields()
```

**Pull request closed without merging**